src/LykkePublicAPI/Modules/ApiModule.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Linq;
+using AspNetCoreRateLimit;
+using Autofac;
+using Autofac.Extensions.DependencyInjection;
+using Common.Log;
+using Core;
+using Core.Domain.Market;
+using Core.Domain.Settings;
+using Core.Services;
+using Lykke.MarketProfileService.Client;
+using Lykke.Service.Assets.Client.Custom;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Caching.Redis;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Services;
+
+namespace LykkePublicAPI.Modules
+{
+    public class ApiModule : Module
+    {
+        private readonly Settings _settings;
+        private readonly PublicApiSettings _apiSettings;
+        private readonly ILog _log;
+        private readonly ServiceCollection _services;
+
+        public ApiModule(Settings settings, ILog log)
+        {
+            _settings = settings;
+            _apiSettings = settings.PublicApi;
+            _log = log;
+
+            _services = new ServiceCollection();
+        }
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            builder.RegisterInstance(_log);
+
+            _services.AddMemoryCache();
+
+            builder.RegisterInstance(_settings);
+            builder.RegisterInstance(_apiSettings);
+            builder.RegisterInstance(_apiSettings.CompanyInfo);
+
+            builder.RegisterType<OrderBookService>().As<IOrderBooksService>();
+            builder.RegisterType< MarketCapitalizationService>().As<IMarketCapitalizationService>();
+            builder.RegisterType<MarketProfileService>().As<IMarketProfileService>();
+            builder.RegisterType<SrvRateHelper>().As<ISrvRatesHelper>();
+
+            RegisterServiceClients(builder);
+            RegisterRedisCache(builder);
```

Comment:
it's not used
https://lykkex.atlassian.net/wiki/spaces/LKEWALLET/pages/177111113/Working+with+Redis
"Oh, seems like it works in-memory"

Reply:
builder.Populate(_services); from line 57 is not that builder.Populate(_services);, which is in the Startup. It not contains services.AddMvc(), so it's fine.